pkg/doc/verifiable/credential_test.go

```diff
@@ -884,34 +900,38 @@ func TestDecodeIssuer(t *testing.T) {
 	})
 
 	t.Run("Decode Issuer identified by empty ID and name", func(t *testing.T) {
-		_, err := decodeIssuer(map[string]interface{}{
+		iID, err := decodeIssuer(map[string]interface{}{
 			"name": "Example University",
 		})
 		require.Error(t, err)
 		require.EqualError(t, err, "issuer ID is not defined")
+		require.Empty(t, iID)
 	})
 
 	t.Run("Decode Issuer with invalid type of ID", func(t *testing.T) {
-		_, err := decodeIssuer(map[string]interface{}{
+		iID, err := decodeIssuer(map[string]interface{}{
 			"id": 55,
 		})
 		require.Error(t, err)
 		require.EqualError(t, err, "value of key 'id' is not a string")
+		require.Empty(t, iID)
 	})
 
 	t.Run("Decode Issuer with invalid type of name", func(t *testing.T) {
-		_, err := decodeIssuer(map[string]interface{}{
+		iID, err := decodeIssuer(map[string]interface{}{
 			"id":   "did:example:76e12ec712ebc6f1c221ebfeb1f",
 			"name": 55,
 		})
 		require.Error(t, err)
 		require.EqualError(t, err, "value of key 'name' is not a string")
+		require.Empty(t, iID)
 	})
 
 	t.Run("Decode Issuer of invalid type", func(t *testing.T) {
-		_, err := decodeIssuer(77)
+		iID, err := decodeIssuer(77)
```

Comment:
Interesting name - iID. Again, still optional!

Reply:
This is really not well readable :)